src/cascadia/UIHelpers/Converters.h

```diff
@@ -34,7 +35,5 @@ namespace winrt::Microsoft::Terminal::UI::implementation
 
 namespace winrt::Microsoft::Terminal::UI::factory_implementation
 {
-    struct Converters : ConvertersT<Converters, implementation::Converters>
-    {
-    };
+    BASIC_FACTORY(Converters);
```

Comment:
ugh converters shouldn't even NEED a factory. I've gotta fix that. It's currently part of #16931 

Would you mind reverting this part until then?